examples/encoding/plot_rerp_encoder.py

```diff
@@ -0,0 +1,67 @@
+"""
+========================================
+Regression on continuous event-related data (rER[P/F])
+========================================
+
+This demonstrates how rER[P/F]s - regressing the continuous data - is a
+generalisation of traditional averaging. If all preprocessing steps
+are the same, no overlap between epochs exists, and all predictors are binary,
+regression is virtually identical to traditional averaging.
+
+If overlap exists and/or predictors are continuous, traditional averaging
+is inapplicable, but regression can still estimate effects.
+
+rERPs are described in:
+
+    Smith, N. J., & Kutas, M. (2015). Regression-based estimation of ERP
+    waveforms: II. Non-linear effects, overlap correction, and practical
+    considerations. Psychophysiology, 52(2), 169-189.
+"""
+# Authors: Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+from mne.encoding.model import EventRelatedRegressor
+from mne.encoding.feature import clean_inputs
+
+# Load and preprocess data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg='grad', stim=True, eeg=False).filter(1, None, method='iir')
+
+# Set up events
+events = mne.find_events(raw)
+event_id = {'Aud/L': 1, 'Aud/R': 2}
+tmin, tmax = -.1, .5
+
+# Regular epoching
+picks = mne.pick_types(raw.info, meg=True)
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, reject=None,
+                    baseline=None, preload=True, verbose=False)
+
+# rERF
+rerp = EventRelatedRegressor(raw, events, est='cholesky', event_id=event_id,
+                             tmin=tmin, tmax=tmax,
+                             preproc_func_xy=clean_inputs)
```

Comment:
>It would be pretty easy to add a continuous_inputs parameter to this class, though it would then be pretty close to the EncodingModel class, so not sure what would be the best way to split them up.

Why split them up? That's my point.

>This is what EventsBinarizer does, no? You can just do EventsBinarizer(n_times).fit_transform(event_ixs, event_ids, event_dict) and it would accomplish this, right?

Yes - though for maximal convenience, this would not have to be user addressed. Ie., dealt with as here.